assets/css/style.css

```diff
@@ -3,40 +3,78 @@
   src: url(../fonts/yeseva-regular.ttf);
 }
 
+.about-cook {
+  transform: translateX(15%);
+}
+
+.about-food {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 430px;
+  text-align: center;
+}
+
+.about-title {
+  color: #cc9900;
+  font-family: 'Yeseva One', 'Arial', 'Times New Roman', serif;
+  font-size: 36px;
+  padding: 20px;
+}
+
+.about-txt {
+  color: #4c4c4c;
+  font-family: 'Lato-Regular', 'Arial', 'Times New Roman', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 20px;
+}
+
 body {
   margin: 0 auto;
 }
 
-body, html, ul, ol, li, p, h1, h2, h3, h4 {
+body,
+html,
+ul,
+ol,
+li,
+p,
+h1,
+h2,
+h3,
+h4 {
   margin: 0px;
   padding: 0px;
 }
 
-div, 
+div,
 button {
- box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 .button {
   display: flex;
   justify-content: center;
   max-width: 486px;
   margin: 0 auto;
-  padding-bottom: 130px;
+  box-sizing: border-box;
```

Comment:
You have it here - https://github.com/thekarinaten/lambda-restaurant/pull/5/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R52